src/utils/pagination/calculateLastPossiblePageNumber.test.ts

```diff
@@ -0,0 +1,20 @@
+import { calculateLastPossiblePageNumber } from "./calculateLastPossiblePageNumber"
+
+describe("calculateLastPossiblePageNumber", () => {
+  it.each([
+    { totalCases: 10, maxPageItems: 10, expected: 1 },
+    { totalCases: 5, maxPageItems: 10, expected: 1 },
+    { totalCases: 10, maxPageItems: 25, expected: 1 },
+    { totalCases: 10, maxPageItems: 50, expected: 1 },
+    { totalCases: 5, maxPageItems: 100, expected: 1 },
+    { totalCases: 11, maxPageItems: 10, expected: 2 },
+    { totalCases: 26, maxPageItems: 10, expected: 3 },
+    { totalCases: 0, maxPageItems: 10, expected: 1 },
+    { totalCases: 0, maxPageItems: 0, expected: 1 }
```

Comment:
The last test scenario would never happen. If `maxPageItems` is set to 0, the user would select to display 0 items per page.